### PR TITLE
Add multiple public French services

### PR DESCRIPTION
**Pull request by Rathmox**
Mon Compte Formation -> easy
Ameli -> Easy
Mon Espace Santé -> Easy
Mes Services Étudiant -> Hard
impots.gouv.fr -> Hard
data.gouv.fr -> easy
France Identité -> Medium
La Poste -> Hard
L'identité Numérique La Poste -> Medium
YRIS -> Medium
Pix -> Hard
Pass Culture -> Easy
CNIL -> Hard
Agence nationale des titres sécurisés -> easy
Service-Public.fr -> easy

### CODE REVIEW

File: _data/sites.json [hunk 1]
```diff
@@ -4771,6 +4810,17 @@
...
     },
 
+    {
+        "name": "data.gouv.fr",
+        "url": "https://doc.data.gouv.fr/gestion-du-compte/supprimer-un-compte/",
+        "difficulty": "easy",
+        "notes": "On your profile page, click on the arrow next to the \"Edit\" button, in the top right of your screen, then on \"Delete my profile\"",
+        "notes_fr": "Sur la page de votre profil, cliquez sur la flèche située à côté du bouton \"Éditer\", en haut à droite de votre écran, puis sur \"Supprimer le profil\"",
+        "domains": [
+            "www.data.gouv.fr"
```
Comment (tupaschoal) [inline _data/sites.json:4820]:
```suggestion
            "data.gouv.fr",
            "doc.data.gouv.fr"
```

File: _data/sites.json [hunk 2]
```diff
@@ -7258,6 +7308,18 @@
         ]
     },
 
+    {
+        "name": "France Identité",
+        "url": "https.//idp.france-identite.gouv.fr/usager",
+        "difficulty": "medium",
+        "notes": "Enter your document number then a deletion code recieved by e-mail to delete your account.",
```
Comment (tupaschoal) [inline _data/sites.json:7315]:
```suggestion
        "difficulty": "easy",
        "notes": "Enter your document number then a deletion code received by e-mail to delete your account.",
```
Sounds easy enough

File: _data/sites.json [hunk 3]
```diff
@@ -9945,6 +10007,17 @@
...
     },
 
+    {
+        "name": "impots.gouv.fr",
+        "url": "https://www.impots.gouv.fr/",
+        "difficulty": "impossible",
+        "notes": "Your account depends on your fiscal number, you cannot delete it.",
+        "notes_fr": "Votre compte dépend de votre numéro fiscal, vous ne pouvez pas le supprimer.",
+        "domains": [
+            "www.impots.gouv.fr"
```
Comment (tupaschoal) [inline _data/sites.json:10017]:
```suggestion
            "impots.gouv.fr"
```

File: _data/sites.json [hunk 4]
```diff
@@ -11152,6 +11225,30 @@
         ]
     },
 
+    {
+        "name": "L'identité Numérique La Poste",
+        "url": "https://aide.lidentitenumerique.laposte.fr/kb/guide/fr/comment-supprimer-mon-identite-numerique-DmTz0vr4Hc/Steps/101590",
+        "notes": "On your customer page, click on the \"My Profile\" icon in the top right corner, then at the bottom on \"Suppression de votre Identité Numérique\". You will need to enter the 4 last digits of the ID card you used when registering.",
+        "notes_fr": "Sur votre espace client, cliquez sur l’icône \"Mon profil\" en haut à droite puis tout en bas sur \"Suppression de votre Identité Numérique\". Vous devrez entrer les 4 derniers chiffres de la carte d'identité utilisée lors de l'inscription.",
+        "difficulty": "medium",
```
Comment (tupaschoal) [inline _data/sites.json:11233]:
```suggestion
        "difficulty": "easy",
```
Also sounds easy

File: _data/sites.json [hunk 5]
```diff
@@ -11152,6 +11225,30 @@
...
+        "domains": [
+            "lidentitenumerique.laposte.fr",
+            "moncompte.laposte.fr"
```
Comment (tupaschoal) [inline _data/sites.json:11236]:
```suggestion
            "aide.lidentitenumerique.laposte.fr"
```

File: _data/sites.json [hunk 6]
```diff
@@ -12782,6 +12879,18 @@
...
     },
 
+    {
+        "name": "Mes Services Étudiant",
+        "url": "https://www.messervices.etudiant.gouv.fr/envole/message/new",
+        "difficulty": "hard",
+        "notes": "You can request the deletion of your account via the assistance form.",
+        "notes_fr": "Vous pouvez faire une demande de suppression de votre compte via le formulaire d'assistance.",
+        "domains": [
+            "www.messervices.etudiant.gouv.fr",
```
Comment (tupaschoal) [inline _data/sites.json:12889]:
```suggestion
            "messervices.etudiant.gouv.fr",
```

File: _data/sites.json [hunk 7]
```diff
@@ -13144,6 +13253,29 @@
...
     },
 
+    {
+        "name": "Mon Compte Formation",
+        "url": "https://www.moncompteformation.gouv.fr/espace-public/comment-me-desinscrire-de-mon-compte-formation",
+        "difficulty": "easy",
+        "notes": "On the \"My Account\" page, click on \"Unregister\"",
+        "notes_fr": "Sur la page \"Mon Compte\", cliquez sur \"Me Désinscrire\"",
+        "domains": [
+            "www.moncompteformation.gouv.fr"
```
Comment (tupaschoal) [inline _data/sites.json:13263]:
```suggestion
            "moncompteformation.gouv.fr"
```

File: _data/sites.json [hunk 8]
```diff
@@ -13144,6 +13253,29 @@
...
+    },
+
+    {
+        "name": "Mon Espace Santé",
+        "url": "https://www.monespacesante.fr/questions-frequentes/acceder-a-mon-espace-sante/14",
+        "difficulty": "easy",
+        "notes": "Login to your account, click on your profile, then on \"Paramètres\". Then click on \"Clôture du profil\"",
+        "notes_fr": "Connectez vous à l'aide de vos identifiants puis cliquez sur votre profil situé en haut à droite de l'écran puis sur \"Paramètres\". Cliquez ensuite sur \"Clôture du profil\"",
+        "domains": [
+            "www.monespacesante.fr",
```
Comment (tupaschoal) [inline _data/sites.json:13274]:
```suggestion
            "monespacesante.fr",
```

File: _data/sites.json [hunk 9]
```diff
@@ -15583,6 +15726,19 @@
         ]
     },
 
+    {
+        "name": "Pix",
+        "url": "https://support.pix.org/en/support/tickets/new",
+        "url_fr": "https://pix.fr/support/form/particulier-fr-fr",
+        "difficulty": "hard",
+        "notes": "You can request that your account be deleted by making a request through our contact form",
```
Comment (tupaschoal) [inline _data/sites.json:15734]:
```suggestion
        "notes": "You can request that your account be deleted by making a request through their contact form",
```

File: _data/sites.json [hunk 10]
```diff
@@ -22843,6 +23011,18 @@
...
 
+    {
+        "name": "YRIS",
+        "url": "https://www.yris.eu/en/frequently-asked-questions/",
+        "url_fr": "https://www.yris.eu/fr/questions-frequentes/",
+        "difficulty": "medium",
+        "notes": "To unsubscribe from YRIS, open the application on your smartphone, log in, go to \"Settings\" (the icon at the bottom right of the main screen), then click on \"Delete my account\". If you are unable to log in, you can request to unsubscribe via the contact form.",
+        "notes_fr": "Pour vous désinscrire d’YRIS, ouvrez l’application sur votre smartphone, connectez-vous, allez dans \"Paramètres\" (l’icône en bas à droite de l’écran principal), puis cliquez sur \"Supprimer mon compte\". Si vous n’arrivez pas à vous connecter, vous pouvez faire une demande de désinscription via le formulaire de contact.",
+        "domains": [
+            "www.yris.eu"
```
Comment (tupaschoal) [inline _data/sites.json:23022]:
```suggestion
            "yris.eu"
```

**Comment by tupaschoal** [comment]
Thanks for all these contributions and the recent PRs!

**Comment by Rathmox** [comment]
you're welcome, I want this project to help most people and when I see services I need that are missing, I directly add them so I help everyone

**Comment by tupaschoal** [comment]
That's the spirit :)

**Pull request merged**